src/BehaviorsSDKManaged/Microsoft.Xaml.Interactions/Core/TypeConverterHelper.cs

```diff
@@ -63,6 +64,15 @@ public static Object Convert(string value, string destinationTypeFullName)
 
             return null;
         }
+        public static Object Convert(string value, Type destinationType)
+        {
+            var typeInfo = destinationType.GetTypeInfo();
+
+            if (typeInfo.IsEnum)
+                return Enum.Parse(destinationType, value);
```

Comment:
Should we use Enum.Parse or Enum.TryParse? This would [mostly] be a developer issue, but if they have the wrong string for the enum this would throw an exception.

Reply:
My suggestion would be to let the exception happen. As a developer I would be happier about my code throwing an exception than it being silently ignoring my configuration and not working.